### ISSUE TITLE
Fail fast on missing curl, don't retry

### ISSUE THREAD

**Issue by sxa** [AdoptOpenJDK/openjdk-tests#1830]
Tried to run a test job on a machine that hadn't had curl instaled - it went into a "retry every 5 minutes loop". The situation where curl simply doesn't exist should ideally be trapped and cause an immediate failure:

```

12:10:53  + ./openjdk-tests/get.sh -s /mnt/jenkinswork/workspace/Grinder -t /mnt/jenkinswork/workspace/Grinder/openjdk-tests -p aarch64_linux -r customized -c https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_aarch64_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz --openj9_repo https://github.com/eclipse/openj9.git --openj9_branch master --tkg_repo https://github.com/AdoptOpenJDK/TKG.git --tkg_branch master
12:10:53  TESTDIR: /mnt/jenkinswork/workspace/Grinder/openjdk-tests
tests
12:10:53  get jdk binary...
12:10:53  _ENCODE_FILE_NEW=UNTAGGED curl -OLJSks  https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_aarch64_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz
12:10:53  ./openjdk-tests/get.sh: line 267: curl: command not found
12:10:53  curl error code: 127. Sleep 300 secs, then retry 1...

12:16:06  check for OpenJDK11U-jdk_aarch64_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz. If found, the file will be removed.
12:16:06  _ENCODE_FILE_NEW=UNTAGGED curl -OLJSks  https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_aarch64_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz
12:16:06  ./openjdk-tests/get.sh: line 267: curl: command not found
12:16:06  curl error code: 127. Sleep 300 secs, then retry 2...

12:21:04  check for OpenJDK11U-jdk_aarch64_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz. If found, the file will be removed.
12:21:04  _ENCODE_FILE_NEW=UNTAGGED curl -OLJSks  https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_aarch64_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz
```

**Comment by smlambert** [comment]
and people keep telling me, 'you don't need this Shelley, we use Ansible'... ;)

Related: https://github.com/AdoptOpenJDK/TKG/issues/45#issuecomment-601497882

I believe a very large percentage of failures we see are infra/config related, TKG/issue 45 will try to address the common ones we see.

**Comment by sxa** [comment]
`<cough>`May be running on one of my personal machines and don't want to run the full suite on it`</cough>`

**Comment by smlambert** [comment]
also, don't run the big set (TARGET=sanity.openjdk), run a fast subset (TARGET=jdk_math), if you want to verify that things are set up right.

**Comment by sxa** [comment]
I've mostly been using `generalSanityTest_0`  target in functional today to get the `SortTest` test, although the overhead of all the git extracts when doing it in Grinder dwarfs the time to run the target somewhat (Switched to running the test manually to speed the process up a bit in some cases!)

**Comment by xichen1** [comment]
Fixed in #2403.
